charts/linkerd2/values.yaml

```diff
@@ -196,6 +196,7 @@ grafana:
   enabled: true
   name: linkerd-grafana
   image: gcr.io/linkerd-io/grafana
+  tag: *linkerd_version
```

Comment:
To have this be more in line with the other settings in this file, could you use `image.name` and `image.version` instead?